packages/react-instantsearch-core/src/connectors/connectAutoComplete.js

```diff
@@ -29,14 +30,25 @@ function getHits(searchResults) {
       searchResults.results.hits &&
       Array.isArray(searchResults.results.hits)
     ) {
-      return searchResults.results.hits;
+      return addAbsolutePositions(
+        addQueryID(searchResults.results.hits, searchResults.results.queryID),
+        searchResults.results.hitsPerPage,
+        searchResults.results.page
+      );
     } else {
       return Object.keys(searchResults.results).reduce(
         (hits, index) => [
           ...hits,
           {
             index,
-            hits: searchResults.results[index].hits,
+            hits: addAbsolutePositions(
```

Comment:
I think they make be more readable of it was split in 
```js
let hits = searchResults.results[index].hits
hits = addQueryID(hits)
hits = addAbsolutePositions(hits)
```

Reply:
could be, but then I would need to change the function to a "block arrow", and didn't feel like that was very necessary here